src/Type/ArrayType.php

```diff
@@ -379,6 +379,11 @@ public function shiftArray(): Type
 		return $this;
 	}
 
+	public function shuffleArray(): Type
+	{
+		return $this;
```

Comment:
I just checked what shuffle() does in PHP documentation. Sounds like it's always gonna make a list - reindexes the keys. Can you check and fix this method in ArrayType? Thanks.

Reply:
yes it does.. thx for checking this, I really shouldn't have done this in such a hurry :)